[2025-10-19T16:19:41.511+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-19T16:19:41.549+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bash_operator_dag.print_system_info manual__2025-10-15T00:39:01.997267+00:00 [queued]>
[2025-10-19T16:19:41.563+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bash_operator_dag.print_system_info manual__2025-10-15T00:39:01.997267+00:00 [queued]>
[2025-10-19T16:19:41.565+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-10-19T16:19:41.587+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): print_system_info> on 2025-10-15 00:39:01.997267+00:00
[2025-10-19T16:19:41.597+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=267) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-19T16:19:41.598+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'bash_operator_dag', 'print_system_info', 'manual__2025-10-15T00:39:01.997267+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/bash_dag.py', '--cfg-path', '/tmp/tmpuw8bbkp9']
[2025-10-19T16:19:41.600+0000] {standard_task_runner.py:63} INFO - Started process 269 to run task
[2025-10-19T16:19:41.601+0000] {standard_task_runner.py:91} INFO - Job 16: Subtask print_system_info
[2025-10-19T16:19:41.681+0000] {task_command.py:426} INFO - Running <TaskInstance: bash_operator_dag.print_system_info manual__2025-10-15T00:39:01.997267+00:00 [running]> on host 95f0e78c2c2c
[2025-10-19T16:19:41.830+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='your_name' AIRFLOW_CTX_DAG_ID='bash_operator_dag' AIRFLOW_CTX_TASK_ID='print_system_info' AIRFLOW_CTX_EXECUTION_DATE='2025-10-15T00:39:01.997267+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-15T00:39:01.997267+00:00'
[2025-10-19T16:19:41.832+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-19T16:19:41.854+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-10-19T16:19:41.856+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'bash /opt/***/dags/data/system_info.sh']
[2025-10-19T16:19:41.873+0000] {subprocess.py:86} INFO - Output:
[2025-10-19T16:19:41.882+0000] {subprocess.py:93} INFO - ===== System Information =====
[2025-10-19T16:19:41.883+0000] {subprocess.py:93} INFO - /opt/***/dags/data/system_info.sh: line 4: data: command not found
[2025-10-19T16:19:41.884+0000] {subprocess.py:93} INFO - Dtate and Time:
[2025-10-19T16:19:41.885+0000] {subprocess.py:93} INFO - /opt/***/dags/data/system_info.sh: line 5: uptime: command not found
[2025-10-19T16:19:41.885+0000] {subprocess.py:93} INFO - Uptime:
[2025-10-19T16:19:41.886+0000] {subprocess.py:93} INFO - Logged in users:
[2025-10-19T16:19:41.902+0000] {subprocess.py:93} INFO - 
[2025-10-19T16:19:41.903+0000] {subprocess.py:93} INFO - ===== Disk Usage Summary =====
[2025-10-19T16:19:41.910+0000] {subprocess.py:93} INFO - Filesystem      Size  Used Avail Use% Mounted on
[2025-10-19T16:19:41.912+0000] {subprocess.py:93} INFO - overlay        1007G   20G  936G   3% /
[2025-10-19T16:19:41.912+0000] {subprocess.py:93} INFO - shm              64M   64K   64M   1% /dev/shm
[2025-10-19T16:19:41.913+0000] {subprocess.py:93} INFO - /dev/sdd       1007G   20G  936G   3% /etc/hosts
[2025-10-19T16:19:41.914+0000] {subprocess.py:93} INFO - C:\             269G  196G   73G  74% /opt/***/dags
[2025-10-19T16:19:41.914+0000] {subprocess.py:93} INFO - 
[2025-10-19T16:19:41.915+0000] {subprocess.py:93} INFO - ===== Top 5 Memory Consuming Processes =====
[2025-10-19T16:19:41.915+0000] {subprocess.py:93} INFO - /opt/***/dags/data/system_info.sh: line 15: ps: command not found
[2025-10-19T16:19:41.917+0000] {subprocess.py:93} INFO - 
[2025-10-19T16:19:41.918+0000] {subprocess.py:93} INFO - ===== Network Interfaces and IPs =====
[2025-10-19T16:19:41.918+0000] {subprocess.py:93} INFO - /opt/***/dags/data/system_info.sh: line 19: ip: command not found
[2025-10-19T16:19:41.922+0000] {subprocess.py:93} INFO - 
[2025-10-19T16:19:41.923+0000] {subprocess.py:93} INFO - Script run completed!
[2025-10-19T16:19:41.924+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-10-19T16:19:41.925+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-19T16:19:41.959+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=bash_operator_dag, task_id=print_system_info, run_id=manual__2025-10-15T00:39:01.997267+00:00, execution_date=20251015T003901, start_date=20251019T161941, end_date=20251019T161941
[2025-10-19T16:19:42.018+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-19T16:19:42.046+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-19T16:19:42.057+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
